<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-I12I50L" name="testScripts.Home.TC_HomePage_Validation" tests="4" failures="1" timestamp="20 Nov 2020 17:42:51 GMT" time="12.889" errors="0">
  <testcase name="testTruckLink" time="0.472" classname="testScripts.Home.TC_HomePage_Validation"/>
  <testcase name="testAutomobileLink" time="0.642" classname="testScripts.Home.TC_HomePage_Validation">
    <failure type="java.lang.AssertionError" message="Automobile link is not present on Home page expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Automobile link is not present on Home page expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at testScripts.Home.TC_HomePage_Validation.testAutomobileLink(TC_HomePage_Validation.java:61)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testAutomobileLink -->
  <testcase name="testMotorCycleLink" time="11.212" classname="testScripts.Home.TC_HomePage_Validation"/>
  <testcase name="testCamperLink" time="0.563" classname="testScripts.Home.TC_HomePage_Validation"/>
</testsuite> <!-- testScripts.Home.TC_HomePage_Validation -->
